spec/index.bs

```diff
@@ -541,7 +541,7 @@ The following special property types can be defined on [=WebGPU objects=]:
 
 : <dfn dfn data-timeline=const>immutable property</dfn>
 ::
-    A read-only slot set during initialization of the object. It can be accessed from any timeline.
+    A read-only slot set during initialization of the object. It can be accessed from [=any timeline=].
```

Comment:
nit: IMO probably doesn't make sense to link to `timeline agnostic` as that's a specific type of algorithm, but immutable properties can be accessed from any type of algorithm.

```suggestion
    A read-only slot set during initialization of the object. It can be accessed from any timeline.
```